XRTK-Core/Packages/com.xrtk.core/Definitions/InputSystem/Simulation/InputSimulationDataProviderConfiguration.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) XRTK. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using UnityEngine;
+using XRTK.Attributes;
+using XRTK.Definitions.Utilities;
+using XRTK.Interfaces.Providers.InputSystem.Simulation;
+
+namespace XRTK.Definitions.InputSystem.Simulation
+{
+    [Serializable]
+    public struct InputSimulationDataProviderConfiguration
```

Comment:
Do we actually need a special configuration type for this?

Reply:
If we are going to keep input simulation data providers separate from controller data providers yes. Because it should only allow providers implementing `IMixedRealityInputSimulationDataProvider` and using `BaseMixedRealityInputSimulationDataProviderProfile `